.travis-ci.sh

```diff
@@ -75,6 +75,16 @@ function build_one {
     echo "End   availability check."
     case $TRAVIS_OS_NAME in
     linux)
+      # we need fresh gcc and binutils, maybe...
+      if [ `opam install --dry-run $pkg | grep -c mirage-entropy-xen` -gt 0 ] ; then
```

Comment:
rather than this, could we add a depext tag to signify that travis should install a newer gcc? Perhaps `["travis"; "gcc48"]` or something suitably specific that is unlikely to be selected by anyone else.  Then `mirage-entropy-xen` is not special cased in the `.travis-ci.sh`